### PR TITLE
Add Sepolia / Base Sepolia support for Base cross-chain

### PR DESCRIPTION
**Pull request by michalinacienciala**
The Goerli testnet currently used by Threshold/Keep for development purposes was planned to become deprecated with the end of year 2023. Although it's already 2024, the testnet is still live, but it will loose validators over the time and may become unusable at some point. The testnet that was created to replace it is called
[Holešky](https://github.com/eth-clients/holesky), however it will take some time until it gets integrated with by some of the projects we rely on. As a solution, we decided to switch first to another testnet that is currently live - Sepolia. This testnet's EOL is planned for 2026, which gives us plenty of time to move to Holešky before Sepolia gets deprecated.

The Goerli -> Sepolia migration means that also L2 testnet based on Goerli (Base Goerli) needs to be migrated to Sepolia-based chain (Base Sepolia). In this PR we're adjusting the config used to deploy testnet contracts in the `@keep-network/tbtc-v2-arbitrum` project. After the update, both deploy to `baseGoerli` and `baseSepolia` networks will be supported (we may remove `baseGoerli` support once we're sure the `baseSepolia` deployment works as expected).

TODO
- [x] Add `external/baseSepolia/BaseWormholeTBTC.json` - in order to get the address we need to perform attestation of the Ethereum Sepolia TBTC token on Base Sepolia (but atm Base Sepolia is not available on the list of target networks on https://wormhole-foundation.github.io/example-token-bridge-ui/#/register)

Ref: https://github.com/threshold-network/solidity-contracts/issues/150

### CODE REVIEW

**Comment by michalinacienciala** [comment]
I've asked Wormhole support if they plan to deploy their contracts on Base Sepolia: https://discord.com/channels/856079283104251944/857225885231611935/1171840544364822628, will let here know once I have some more info.

File: cross-chain/base/deploy_l2/14_update_with_polygon_in_wormhole_gateway_mapping.ts [hunk 1]
```diff
@@ -11,7 +11,9 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     "0x1af5DC16568EFF2d480a43A77E6C409e497FcFb9"
 
   // See https://book.wormhole.com/reference/contracts.html
-  // This ID is valid for both Polygon Testnet (Mumbai) and Mainnet
+  // This ID is valid for both Polygonn Goerli-based Testnet (Mumbai) and
+  // Mainnet. Wormhole does not support the Sepolia-based Amoy Testnet yet.
+  // TODO: Update the ID once the support is added.
   const polygonWormholeChainID = 5
```
Comment (michalinacienciala) [inline cross-chain/base/deploy_l2/14_update_with_polygon_in_wormhole_gateway_mapping.ts:17]: Not sure how the deploy on `baseSepolia` will behave when the `polygonWormholeChainID` does point to the value which is good for Mumbai and Polygon Mainnet, but not for Sepolia--based Amoy... Should I skip this script for now?

Comment (michalinacienciala) [reply cross-chain/base/deploy_l2/14_update_with_polygon_in_wormhole_gateway_mapping.ts:17]: Until new Polygon testnet is supported by Wormhole, we can skip this script during deploy.

**Comment by michalinacienciala** [comment]
Tested on a local deployment of the Wormhole portal (`http://localhost:3000/#/transfer`, deployed from https://github.com/michalinacienciala/michalinacienciala.github.io).

Ethereum Sepolia -> Base Sepolia (0.05 tBTC successfully bridged):
* Send tx: https://sepolia.etherscan.io/tx/0x5205093fefa0dde3692660b338ae5a0bb16c3f711c1c4ea48704b8636be04b98
* Warp tx: https://sepolia.basescan.org/tx/0xd232ee8a14ad525d61de0e8c836d98d24db5e0fcc2009409b959c8b28699d08a

Base Sepolia -> Ethereum Sepolia (0.05 tBTC successfully bridged):
* Send tx: https://sepolia.basescan.org/tx/0x273a55714857ab8c431e8b9121265d02234c98f82aab201f1cf3660d2cedd027
* Warp tx: 
https://sepolia.etherscan.io/tx/0xfc62d0491074164cd48b1c8a1a4c49691c54115608e9eb20adeba707e20e6fe9

**Comment by michalinacienciala** [comment]
We've deployed the contracts on Base Sepolia and published a package: [@keep-network/tbtc-v2-base@1.1.0-sepolia.0](https://www.npmjs.com/package/@keep-network/tbtc-v2-base/v/1.1.0-sepolia.0)

We've registered gateways of other L2s in Wormhole Portal and tested bridging between the networks:

Sepolia Base > Sepolia Arbitrum
* https://sepolia.basescan.org/tx/0xc2c9e5963e9454fdd042e716885211b648143b6c569758760dfe05a4ccc67b41
* https://sepolia.arbiscan.io/tx/0xe80fc617e43138102a1ae9a09cf4cc51e16a79f1a1e274b535be880fcecbe681

Sepolia Arbitrum > Sepolia Base
* https://sepolia.arbiscan.io/tx/0x8e0fac7259d0d257e8c13432a3adab075e3926ba421f4a0adbb0cbee63b57aa5
* https://sepolia.basescan.org/tx/0xe87f828a9217ff0ae72651423f74ad106f7e0f5780cf9b1e8bbb37e2e887bc48

Sepolia Base > Sepolia Optimism
* https://sepolia.basescan.org/tx/0xaaed6c5f34bbeecc5abad7a980a123b539abf6e0d5f85c6d15f90f12fd865f8a
* https://sepolia-optimism.etherscan.io/tx/0x5fb740270f77f160cc24df6f97ee0a84917514f87392bdd3c3e9173ebb77cb3d

Sepolia Optimism > Sepolia Base
* https://sepolia-optimism.etherscan.io/tx/0x78bf7eef6cd68006ab428c8353f93252c3753b7704b96d6ce8712a2295456515
* https://sepolia.basescan.org/tx/0xf644eb4659b97f9950b968a16e50e9da66e05e7725323a09d3df47f0ed97b122

**Review by lukasz-zimnoch — APPROVED** [review]

**Pull request merged**